libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -0,0 +1,27 @@
+package com.breezsdk;
+
+import breez_sdk.*;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+class BreezSDKModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {  
+    override fun getName(): String {
+        return "BreezSDK";
+    }
+
+    @ReactMethod
+    fun mnemonicToSeed(mnemonic: String, promise: Promise) {
+        try {
+            var seed = mnemonicToSeed(mnemonic);
+            var response = writableMapOf("type" to "seed", "data" to seed);
```

Comment:
What is the advantage of adding the "type" key here? What is the advantage of this structure vs returning ReadableArray as a single value?

Reply:
I intend to change this to just map the SDK responses as much as possible.